docs/source/index.rst

```diff
@@ -21,7 +21,8 @@
 
     import pymc3 as pm
 
-    X, y = linear_training_data()
+    X, y = pm.math._linear_training_data()
+
```

Comment:
I don't think we need to import numpy. Whoever makes it to this point can infer that by themselves.
```suggestion
    X = np.random.normal(size=100)
    y = np.random.normal(X) * 1.2

```